src/haz3lschool/SchoolExercise.re

```diff
@@ -729,6 +732,10 @@ module F = (ExerciseEnv: ExerciseEnv) => {
       simple_result: ModelResult.simple,
     };
   };
+
+  /* Extracts and returns the DHExp expression
```

Comment:
"Given the evaluation results, collects the relevant information for producing dynamic feedback"